packages/sirius-web/backend/sirius-web-sample-application/src/main/java/org/eclipse/sirius/web/sample/papaya/view/classdiagram/CDCreateClassNodeToolProvider.java

```diff
@@ -45,15 +45,16 @@ public NodeTool create(PapayaViewCache cache) {
         setValue.setValueExpression("aql:'Class'");
 
         var classNodeDescription = cache.getNodeDescription(CDClassNodeDescriptionProvider.NAME);
-        var createView = ViewFactory.eINSTANCE.createCreateView();
-        createView.setParentViewExpression("aql:selectedNode");
-        createView.setSemanticElementExpression("aql:newInstance");
-        createView.setElementDescription(classNodeDescription);
-
-        nodeTool.getBody().add(createInstance);
-        createInstance.getChildren().add(changeContext);
-        changeContext.getChildren().add(setValue);
-        setValue.getChildren().add(createView);
+        if (classNodeDescription.isPresent()) {
```

Comment:
Small nitpick but it would be better to rename variables like this one to `optionalXxx` to match our naming conventions and indicate to everybody the type of the variable